Jenkinsfile

```diff
@@ -12,7 +12,8 @@ void buildOnYoctoNode(String variant, String image) {
     node("Yocto") {
 
         checkout scm
-        def customImage = docker.image("pelux/pelux-yocto:yoctouser")
+        //def customImage = docker.image("pelux/pelux-yocto:yoctouser")
```

Comment:
remove this?

Reply:
It's just for test to not rebuild image on Dockerhub. Will be deleted later